docs/content/_index.md

```diff
@@ -0,0 +1,116 @@
++++
+title = ''
+date = 2024-04-06T22:34:36+05:30
+draft = false
+toc = true
+[cascade]
+    type = 'docs'
++++
+
+# Documentation for `hugo-python-distributions`
+
+This webpage is meant to serve the documentation for the `hugo-python-distributions` project, which aims to provide a distribution
+channel for the extended version of the Hugo static site generator through `pip`-installable binaries (wheels) hosted on the [Python Package Index (PyPI)](https://pypi.org/).
+
+## Table of contents
+
+- [Quickstart](#quickstart)
+- [Supported platforms](#supported-platforms)
```

Comment:
```suggestion
- [Supported platforms](supported-platforms/)
```
This redirect is not working on the page, is this intended?

Reply:
Thanks for flagging this. I missed doing this, I think we should add a link checker soon.